runtime/queries/janet/highlights.scm

```diff
@@ -1 +1,72 @@
-; inherits: clojure
+(kwd_lit) @string.special.symbol
+
+(str_lit) @string
+
+(long_str_lit) @string
+
+(buf_lit) @string
+
+(long_buf_lit) @string
+
+(num_lit) @constant.numeric
+
+[(bool_lit) (nil_lit)] @constant.builtin
+
+(comment) @comment
+
+(quote_lit
+ (sym_lit) @symbol)
+
+(qq_lit
+ (sym_lit) @symbol)
```

Comment:
```suggestion
 (sym_lit) @string.special.symbol)

(qq_lit
 (sym_lit) @string.special.symbol)
```

Reply:
I guess what's being suggested is that `symbol` doesn't exist so use something else?

Assuming so, that makes sense to me, though `string.special.symbol` is used for `kwd_lit` (and according the [syntax highlighting section](https://github.com/helix-editor/helix/blob/84e24b33dcda16d1d64805f34dcc02d82d0de8f1/book/src/themes.md#syntax-highlighting) something similar is Clojure keywords) and a distinction might be nice.

May be just dropping highlighting for `quote_lit` and `qq_lit` for now is another path.  Possibly there isn't much value in highlighting quoted symbols in these types of situations.